doc/specs/#653 - Quake Mode/#653 - Quake Mode.md

```diff
@@ -0,0 +1,629 @@
+a number of pixels per second. In my opinion, that would be harder for users to
+use correctly. I believe that it's easier for users to mentally picture "I'd
+like the dropdown to last 100ms" vs "My monitor is 1504px tall, so I need to set
+this to 15040 to make the window traverse the entire display in .1s"
+
+Some users might want to be able to use the global hotkey to hide the window
+when the window is already visible. This would let the hotkey act as a sort of
+global toggle for the Terminal window. Others might not like that behavior, and
+just want the action to always bring the Terminal into focus, and do nothing if
+the terminal is already focused. To facilitate both these use cases, we'll add
+the following property:
+
+* `"hideWhenVisible": bool`
+  - When `true`: (_default_) When this hotkey is pressed, and the terminal
+    window is currently active, minimize the window.
+  - When `false`: When this hotkey is pressed, and the terminal window is
+    currently active, do nothing.
+
+### Minimize to Tray
+
+Many users have requested that the terminal additionally supports minimizing the
+window "to the tray icon". This is a bit like when you close the Teams window,
+but Teams is actually still running in the system tray, or the "notification
+area".
+
+![The Teams tray icon](tray-icon-000.png)
+
+_fig 1: an example of the Teams tray icon in the notification area_.
+
+When users want to be able to "minimize to the tray", they want:
+* The window to no longer appear on the taskbar
+* The window to no longer appear in the alt-tab order
+
+When minimized to the tray, it's almost as if there's no window for the Terminal
+at all. This can be combined with the global hotkey to quickly restore the window.
+
+The tray icon could be used for a variety of purposes. As a simple start, we
+could include the following three options:
+
+```
+Focus Terminal
+---
+Windows > Window 1 - <un-named window>
+          Window 2 - "This-window-does-have-a-name"
+---
+Quit
+```
+
+Just clicking on the icon would summon the recent terminal window. Right
+clicking would show the menu with "Focus Terminal", "Windows" and "Quit" in it, and
+"Windows" would have nested entries for each Terminal window.
+
+* "Focus Terminal" would do just that - summon the most recent terminal window,
+  wherever it is.
+* "Windows" would have nested popups for each open Terminal window. Each of
+  these nested entries would display the name and ID of the window. Clicking
+  them would summon that window (wherever it may be)
+* "Quit" would be akin to quit in browsers - close all open windows
+  <sup>[[1]](#footnote-1)</sup>.
+
+The tray notification would be visible always when the user has
+`"minimizeToTray": true` set in their settings. If the user has that set to
+false, but would still like the tray, they can specify `"alwaysShowTrayIcon":
+true`. That will cause the tray icon to always be added to the system tray.

```

Comment:
> `"alwaysShowTrayIcon": true`

We could probably do without this setting. Just always show the tray icon.

This gets rid of the weird combination of settings below.

Reply:
From review: some people hate tray clutter, but this might be a cool feature to enable conditionally.